### ISSUE TITLE
Modal's fail after 5 seconds

### ISSUE THREAD

**Issue by DataTraveler1** [ironmansoftware/powershell-universal#1040]
Credit to [Fox_Zoulou](https://forums.ironmansoftware.com/u/Fox_Zoulou) for posting about this & providing sample code on the forums [here](https://forums.ironmansoftware.com/t/problem-modal-in-dynamic-region/6748/2)

#

### Steps to Reproduce

1. Create a dashboard per below code

```
 New-UDDashboard -Title "Title" -Content {
    $object = '' | select title
    $object.title = 'Titletest'
    $TableCSV = @()
    $TableCSV += $object
    New-UDDynamic -Id 'data' -Content {
        $Columns = @(
            New-UDTableColumn -Property title  -Width 300
            New-UDTableColumn -Property rowid -Title Action -Render {
                New-UDButton -Id "testBTN" -Text "testBTN" -OnClick { 
                    #Show-UDToast -Message $eventdata.rowid 
                    Show-UDModal -Content {    
                        New-UDTypography -Text ("confirm action ?")
                    } -Footer {
                        New-UDButton -Text "Confirm" -OnClick { 
                            Hide-UDModal
                            Sync-UDElement -Id 'data' 
                        } 
                        New-UDButton -Text "Cancel" -OnClick { Hide-UDModal }                    
                    }
                }
            }
        )
        New-UDTable -Title "List" -Data $TableCSV -column $Columns -Search
    }  -AutoRefreshInterval 5 -AutoRefresh 
}
```

2. Click the button
3. Wait at least 5 seconds
4. Neither of the buttons will produce any effect when clicked

Note: If you wait less than 5 seconds then it should work.

This issue appears to be related to a timeout refresh?

Changing the value of _JobHandshakeTimeout_ from 5 to 10 did not help

### Expected behavior

```console
The button should still work after 5 seconds
```

### Actual behavior

```console
The button stops working after the first 5 seconds
```

### Environment data

PSU 2.9.1

**More**

When comparing the debug logs between a working button press and a non-working (by following these steps to reproduce) there is the below block of activity in the "working" log versus the non-working.

> 2022-03-10 18:49:43.210 -05:00 [DBG] Received hub invocation: InvocationMessage { InvocationId: "2", Target: "clientEvent", Arguments: [ 62fa6492-f370-43a8-841b-6c6869a38fd5, 62fa6492-f370-43a8-841b-6c6869a38fd5, ,  ], StreamIds: [  ] }.
> 2022-03-10 18:49:43.210 -05:00 [DBG] ClientEvent 62fa6492-f370-43a8-841b-6c6869a38fd5 62fa6492-f370-43a8-841b-6c6869a38fd5
> 2022-03-10 18:49:43.211 -05:00 [DBG] AuthenticationScheme: Cookies was successfully authenticated.
> 2022-03-10 18:49:43.211 -05:00 [DBG] AuthenticationScheme: Cookies was successfully authenticated.

### Visuals

![image](https://user-images.githubusercontent.com/537287/157780780-f1192db7-0c62-4edc-ad19-4d063c3bf74f.png)

**Comment by adamdriscoll** [comment]
This issue has been mentioned on **Ironman Software Forums**. There might be relevant details there:

https://forums.ironmansoftware.com/t/problem-modal-in-dynamic-region/6748/4

**Comment by adamdriscoll** [comment]
This was resolved in 3.1.
